common/src/main/java/org/apache/comet/parquet/ColumnReader.java

```diff
@@ -207,6 +214,7 @@ public CometDecodedVector loadVector() {
       DictionaryEncoding dictionaryEncoding = vector.getField().getDictionary();
 
       CometPlainVector cometVector = new CometPlainVector(vector, useDecimal128);
+      cometVector.setIsUuid(isUuid);
```

Comment:
Why not put it into constructor parameter list?

Reply:
added `isUuid` in constructor parameter list